recipe-page.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE>
+<html>
+  <head>
+    <link href='http://fonts.googleapis.com/css?family=Abel|Yellowtail|Droid+Serif' rel='stylesheet' type='text/css'>
+    <title>Delicious Recipes</title>
+    <link href="style.css" type="text/css" rel="stylesheet" />
+  </head>
+  <div class='container'>
+    <body>
+      <header>
+        <img src='challenge/logo.png' />
+        <section class='headerTextSection'>
+          <h1>Delicious</h1>
+          <p>THE BEST FOOD BLOG ON THE WEB</p>
+        </section>
+        <section class='header-rss-mailSection'>
```

Comment:
It appears you're using is section to add the visual separation of the social icons. Instead, you can use the :nth pseudo selector to achieve that separation.